cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/monitor/RuntimeClientService.java

```diff
@@ -144,6 +144,10 @@ long getProgramFinishTime() {
     return programFinishTime;
   }
 
+  public void onProgramStopRequested(Runnable stopper) {
```

Comment:
should we keep this package private? If not it should add javadoc for public method.

Reply:
The DefaultRuntimeJob is outside of this package. Added a doc